packages/encrypted-blockstore/src/loader.ts

```diff
@@ -433,7 +444,45 @@ export class Loader {
   }
 }
 
+// duplicated in @fireproof/connect
```

Comment:
maybe this means the modules should be one module? not making that change now. Maybe there are other ways to clean up this type dependency graph